.test-infra/jenkins/job_00_seed.groovy

```diff
@@ -104,18 +104,6 @@ job('beam_SeedJob') {
   }
 
   steps {
-    shell {
-      command("""
-        ( cd .test-infra/jenkins/committers_list_generator &&
-        python3.8 -m venv ve3 && source ve3/bin/activate &&
-        pip install --retries 10 --upgrade pip setuptools wheel &&
-        pip install --retries 10 -r requirements.txt &&
-        python main.py -o .. &&
-        deactivate ) ||
-        { echo "ERROR: Failed to fetch committers"; exit 3; }
-      """)
-      unstableReturn(3)
-    }
```

Comment:
Instead of killing everything in committers_list_generator, can we just get rid of this section for now?